### PR TITLE
BaSP-TG-64: Admins List

### PR DESCRIPTION
**Pull request by GuidoCerioni**

### CODE REVIEW

File: src/Components/Admin/index.js [hunk 1]
```diff
@@ -1,115 +1,7 @@
...
-import Input from './Input';
+import styles from './admins.module.css';
 
-function AdminForm() {
-  const [firstName, setFirstName] = useState('');
-  const [lastName, setLastName] = useState('');
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
+const AdminForm = () => {
+  return <section className={styles.container}>yjgjhghjjh</section>;
```
Comment (AlejandroSantini) [inline src/Components/Admin/index.js:4]: What is this return?

**Review by AlejandroSantini — COMMENTED** [review]
General class names like .container should be changed

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,38 +1,87 @@
...
-      <a className={styles.add} href="/admins/form">
-        &#10010; Add New Admin
-      </a>
-      <List list={list} setList={setList} deleteAdmin={deleteAdmin} />
+      {loading ? (
+        <LoadingScreen />
+      ) : (
+        <>
+          <Modal
+            modalTitle="Are you sure you want to delete?"
```
Comment (AlejandroSantini) [inline src/Components/Admins/index.js:68]: You can add paragraph for the text in modalTitle, and in modalTitle put 'Admin Delete'

Comment (AlejandroSantini) [inline src/Components/Admins/index.js:68]: You can add paragraph for the text in modalTitle, and in modalTitle put 'Admin Delete'

**Review by AlejandroSantini — APPROVED** [review]
Nice job Guido!

**Review by AlejandroSantini — CHANGES_REQUESTED** [review]
Correct comments

File: src/Components/Admin/index.js (hunk 1 above)
Comment (GuidoCerioni) [reply src/Components/Admin/index.js:4]: Oh sorry, i made it so i can test if my edit button was working. This file will be deleted after merge.

Comment (AlejandroSantini) [reply src/Components/Admin/index.js:4]: Okey

**Review by AlejandroSantini — APPROVED** [review]
Nice job!!

**Review by Facund0Fernandez — COMMENTED** [review]
Everything seems to work fine. It only remains to correct the breakage of the style when performing a search for something that does not exist in the Search component.

**Review by Facund0Fernandez — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Review by Facund0Fernandez — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Review by AlejandroSantini — APPROVED** [review]

**Pull request merged**